### ISSUE TITLE
Automatically relocate songs

### ISSUE THREAD

**Issue by dspacejs** [Taiko2k/TauonMusicBox#570]
I reinstalled my OS. My music files have moved from another internal hard drive (`/media/daniel/Extra Storage`), to my `/home/daniel/Music` folder.

But all my playlists are using the old path, so they're broken now.

Is there a way to automatically sync my music to the new location? The files are the same, the paths have just changed.

**Comment by Taiko2k** [comment]
Its currently possible by:

1. Exporting each playlist as xspf.
2. Import new location files to a new playlist
3. Do a menu > database > Remove missing tracks
4. Re import the xspf files by drag and drop

Probably need to add a feature named something like "Refind lost tracks" so that this process is simpler

**Comment by dspacejs** [comment]
@Taiko2k yeah that feature would be really useful. Doing this manually is a pain

**Comment by tehmasterer** [comment]
Any update on this request? This is a pretty important feature for a music player, IMO. I would add that preserving metadata (play count, stars) when relinking the paths should be considered too.

**Comment by sstark** [comment]
@tehmasterer My music library is rooted in a single folder called "Music" and several playlists are based on it. I had to migrate my library to another disk, which changed the absolute path to it. To fix tauon, I used the following steps:

- Delete the main playlist "Music" (but keep the others)
- Drag and drop the "Music" folder from its new location into Tauon
- Regenerate other playlists (not even sure that was necessary)

That worked fast and smooth and my play counts and favorites are still there.
